client/webserver/site/src/html/bodybuilder.tmpl

```diff
@@ -104,7 +104,7 @@
 {{end}}
 
 {{define "bottom"}}
-<script src="/js/entry.js?v=88986a77|0cdffb94"></script>
+<script src="/js/entry.js?v=ca7f3afe|a4fa1225"></script>
 </body>
```

Comment:
I rebased and used the ` ./client/webserver/site/bust_caches.sh ` script but I don't know why it's still failing.